index.d.ts

```diff
@@ -0,0 +1,58 @@
+declare module 'waveform-data' {
+
+    type Options = {
+        /* optional props to override defaults */
+        scale?: number,
+        amplitude_scale?: number,
+        split_channels?: boolean
+    }
+
+    export type WaveformAudioContextOptions = {
+        audio_context: AudioContext
+        array_buffer: ArrayBuffer
+    } & Options;
+
+    export type WaveformAudioBufferOptions = {
+        audio_buffer: AudioBuffer
+    } & Options
+
+    export type WaveformFromAudioCallback = (error: null | DOMException, instance?: WaveformData, audioBuffer?: AudioBuffer) => void
+
+    export type AudioWaveform = {
```

Comment:
Do you mind if we name this `JsonWaveformData` - because we have the same type in Peaks.js: https://github.com/bbc/peaks.js/blob/master/peaks.js.d.ts#L320. Sorry, I should have mentioned this before.

Reply:
No problem.